doc/install.md

```diff
@@ -3,200 +3,286 @@
+   `pg1` created in the previous step.
+
+    ```yaml
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: pause
+    spec:
+      replicas: 2
+      selector:
+        matchLabels:
+          app: pause
+      template:
+        metadata:
+          labels:
+            app: pause
+            pod-group.scheduling.sigs.k8s.io: pg1
+        spec:
+          containers:
+          - name: pause
+            image: k8s.gcr.io/pause:3.2
+    ```
+
+1. As PodGroup `pg1` requires at least 3 pods to be scheduled all-together, and there are only 2 Pods
+   so far, so it's expected to observer they are pending:
+
+    All nginx pods are expected to be `Pending` as they cannot be co-scheduled altogether.
+
+    ```bash
+    $ kubectl get pod
+    NAME                     READY   STATUS    RESTARTS   AGE
+    pause-58f7d7db67-7sqgp   0/1     Pending   0          9s
+    pause-58f7d7db67-jbmfv   0/1     Pending   0          9s
    ```
 
+1. Now let's scale the deployment to let it qualify for `minMember` (i.e., 3) of the associated PodGroup:
+
+    ```bash
+    $ kubectl scale deploy pause --replicas 3
+    deployment.apps/pause scaled
+    ```
+
+    And wait for a couple of seconds, it's expected to see all Pods get into running state:
+
+    ```bash
+    $ kubectl get pod
+    NAME                     READY   STATUS    RESTARTS   AGE
+    pause-58f7d7db67-7sqgp   1/1     Running   0          19h
+    pause-58f7d7db67-jbmfv   1/1     Running   0          19h
+    pause-58f7d7db67-ssc8l   1/1     Running   0          19h
+    ```
+
+1. You can also get the PodGroup's spec via:
+
+    ```bash
+    $ kubectl get podgroup pg1 -o yaml
+    apiVersion: scheduling.sigs.k8s.io/v1alpha1
+    kind: PodGroup
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/last-applied-configuration: |
+          {"apiVersion":"scheduling.sigs.k8s.io/v1alpha1","kind":"PodGroup","metadata":{"annotations":{},"name":"pg1","namespace":"default"}, "spec":{"minMember":3,"scheduleTimeoutSeconds":10}}
+      creationTimestamp: "2021-03-16T00:22:34Z"
+      generation: 1
+      managedFields:
+      ...
+      name: pg1
+      namespace: default
+      resourceVersion: "135603"
+      selfLink: /apis/scheduling.sigs.k8s.io/v1alpha1/namespaces/default/podgroups/pg1
+      uid: b4ac3562-54ab-4c1e-89bb-541a81c6acce
+    spec:
+      minMember: 3
+      scheduleTimeoutSeconds: 10
+    status:
+      phase: pending
+    ```
+    
+> ⚠ NOTE: There are some UX issues need to be addressed in controller side -
+> [#166](https://github.com/kubernetes-sigs/scheduler-plugins/issues/166).
+
+## Install release v0.18.9 (old version) and use Coscheduling
+
+If you're running at v0.18.9, which doesn't depend on PodGroup CRD, you should refer to the
+[install doc](https://github.com/kubernetes-sigs/scheduler-plugins/blob/release-1.19/doc/install.md) in
+branch `release-1.19`.
```

Comment:
```suggestion
[install doc](https://github.com/kubernetes-sigs/scheduler-plugins/blob/release-1.18/doc/install.md) in
branch `release-1.18`.
```

Reply:
Thanks for the catch. I put this paragraph into a section `Instal lower-version releases`.